openapi/openapi.yml

```diff
@@ -11,6 +11,25 @@ paths:
     post:
       summary: Get sponsored content
       description: Get a list of spocs based on region and pocket_id from AdZerk. The IP address is used to deduce a rough geographic region, for example "Texas" in the U.S. or "England" in the U.K. The IP is not stored or shared with AdZerk to preserve privacy.
+      parameters:
+        - in: query
+          name: site
+          schema:
+            type: integer
+          required: false
+          description: override siteId in ad decision requests
+        - in: query
+          name: region
+          schema:
+            type: integer
```

Comment:
I think this should be a string? Same for country.

Reply:
! nice catch. a look around to confirm this was right made me realize that the way the query params are parsed implies that these three params are *also* valid params in the request body, so adding them to that part of the spec too.